dashboard/test/ui/features/gdpr_dialog.feature

```diff
@@ -0,0 +1,36 @@
+@dashboard_db_access
+
+Feature: GDPR Dialog - data transfer agreement
+
+  Scenario: EU user sees the GDPR Dialog on dashboard, opt out
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then I wait to see ".ui-test-gdpr-dialog"
+    Then I wait to see ".ui-test-gdpr-dialog-logout"
+    Then I click selector ".ui-test-gdpr-dialog-logout"
+    Then check that I am on "http://code.org/"
```

Comment:
Maybe we could make sure that the dialog is *not* showing when we're now logged out?

Reply:
This is proving to be tricky because `GDPRDialog` isn't hidden or not visible based on how our UI steps are looking for it.  The dialog is there in the DOM and being "seen" it's just closed.   Is there a way to say something more like "does this element have these props at this time in a UI test"?

Reply:
since the dialog blocks the user from clicking on the anything on the page,  I feel comfortable using the success of the user doing something on the page as a good indication that the dialog is not visible

Reply:
We appear to have "is visible" functionality which might help.  

If we're relying on the user doing things on the page, we have to make sure that's not just because we're using JS to interact with elements directly when a real user wouldn't be able to get to them.